app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours_signed/CheckOpeningHoursSigned.kt

```diff
@@ -33,7 +33,10 @@ class CheckOpeningHoursSigned (
             or older today -1 years
           )
           and access !~ private|no
-          and (name or brand or noname = yes or name:signed = no or amenity = recycling)
+          and (
+            name or brand or noname = yes or name:signed = no
+            or amenity = recycling|toilets|bicycle_rental or leisure=park or barrier
```

Comment:
Shouldn't we drop recycling from here too?

Also should this either be `~`, or can the one in AddOpeningHours be `=` too?

Reply:
Good catches!

Fixed `=` vs `~`

`amenity = recycling and recycling_type = centre` should be either added to name quest or allowed to have no name in opening hours quest, not checked which would be preferable

Reply:
https://wiki.openstreetmap.org/wiki/Tag:recycling_type%3Dcentre

> a public recycling centre

So asking for opening hours is fine